### ISSUE TITLE
TypeError: Class extends value undefined is not a constructor or null

### ISSUE THREAD

**Issue by xmlking** [HoudiniGraphql/houdini#856]
### Describe the bug

I am using `houdini-plugin-svelte-global-stores` , and when I test this example project, getting the following error
```
TypeError: Class extends value undefined is not a constructor or null
    at /$houdini/plugins/houdini-svelte/stores/listPokemon.js:4:39
    at async instantiateModule (file:///Users/schintha/Developer/Work/SPA/bbb/houdini-issues/node_modules/.pnpm/vite@4.0.4/node_modules/vite/dist/node/chunks/dep-5e7f419b.js:52224:9)

```

### Severity

annoyance

### Steps to Reproduce the Bug

https://github.com/xmlking/houdini-issues
pnpm i
pnpm dev
open http://localhost:5173/test1

### Reproduction

https://github.com/xmlking/houdini-issues

**Comment by xmlking** [comment]
sorry , now the repo change to public.

**Comment by xmlking** [comment]
any thoughts on why I am getting 
`ReferenceError: Cannot access 'QueryStoreOffset' before initialization`

<img width="584" alt="image" src="https://user-images.githubusercontent.com/627951/215625923-41788c77-dd40-4e7c-afed-1cd5f36cca14.png">

I am testing simple API https://github.com/xmlking/svelte-starter-kit/tree/main/src/routes/(play)/play/fetching 
page show data first and within sec display this error message...

**Comment by AlecAivazis** [comment]
This is with the `@next` version, right?

**Comment by AlecAivazis** [comment]
I was able to reproduce and will have a fix published very soon

**Comment by AlecAivazis** [comment]
This should be fixed in `v1.0.0-next.7`
